eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSMergeWriter.java

```diff
@@ -140,16 +166,9 @@ private void insertMergeOperation(ClusteredVariantEntity mergeDestination, Clust
     private void insertRSRecordForMergeDestination(ClusteredVariantEntity mergeDestination) {
         Class<? extends ClusteredVariantEntity> clusteredVariantCollectionToWriteTo =
                 clusteringWriter.getClusteredVariantCollection(mergeDestination.getAccession());
```

Comment:
Requisite check is done inside the merge method. So no need to repeat that here.